src/react/queries.ts

```diff
@@ -83,4 +83,12 @@ export const getUserListUsersQuery = (accountName: string, IAMClient: IAMClient)
   enabled: IAMClient !== null,
   refetchOnMount: false,
   refetchOnWindowFocus: false,
+});
+
+export const getUserPoliciesQuery = (UserName: string, IAMClient: IAMClient) => ({
```

Comment:
The name is bit misleading. I would suggest to rename
```suggestion
export const getPoliciesQuery = (UserName: string, IAMClient: IAMClient) => ({
```